integration_tests/sdk/setup_integration.py

```diff
@@ -134,8 +134,13 @@ def _setup_mongo_db_data(client: Client, mongo_db: MongoDBIntegration) -> None:
 
 def _setup_relational_data(client: Client, db: RelationalDBIntegration) -> None:
     # Find all the tables that already exist.
-    existing_table_names = set(db.list_tables()["tablename"])
-
+    columns = list(db.list_tables().columns)
+    if "tablename" in columns:
+        existing_table_names = set(db.list_tables()["tablename"])
+    elif "table_name" in columns:
```

Comment:
How hard would it be to edit the mariaDB list_tables() to return the tables in a "tablename" column? I'd much rather make that api consistent then branch in our testing logic.

Reply:
Easier than expected! Updated.